### PR TITLE
More lenient label input

### PR DESCRIPTION
**Pull request by andreavs**

### CODE REVIEW

File: cognite/client/data_classes/assets.py [hunk 1]
```diff
@@ -103,8 +100,7 @@ def _load(cls, resource: Union[Dict, str], cognite_client=None):
         if isinstance(resource, Dict):
             if instance.aggregates is not None:
                 instance.aggregates = AggregateResultItem(**instance.aggregates)
-        if instance.labels is not None:
-            instance.labels = [Label._load(label) for label in instance.labels]
+        instance.labels = Label._load_list(instance.labels)
```
Comment (sanderland) [inline cognite/client/data_classes/assets.py:103]: this seems a bit meh, either Label._load or LabelList._load?

Comment (sanderland) [reply cognite/client/data_classes/assets.py:103]: if LabelList is already a thing that is

Comment (andreavs) [reply cognite/client/data_classes/assets.py:103]: agree it does not really belong perfectly in this place - initially tried putting it in utils._auxillary, but it caused a nasty circular dependency issue that I wasn't able to sort out :cry:

**Review by sanderland — APPROVED** [review]
:1st_place_medal:

**Pull request merged**